src/OrchardCore.Modules/OrchardCore.Templates/Views/Template/Edit.cshtml

```diff
@@ -19,7 +19,7 @@
 
 @if (Model.AdminTemplates)
 {
-    <zone Name="Title"><h1>@RenderTitleSegments(T["Edit Admin Template"])</h1></zone>
+    <zone Name="Title"><h1 class="display-inline float-left">@RenderTitleSegments(T["Edit Admin Template"])</h1></zone>
```

Comment:
Why this for edit, but not create?

I don't think you actually need this at all anymore, as it is all in a card right?

Reply:
Seeems there are not needed anymore, let me double check and confirm